perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -1477,6 +1477,11 @@ sub process_garden_renew : Private {
     $c->forward('setup_garden_sub_params', [ $data, $type ]);
     $c->forward('add_report', [ $data, 1 ]) or return;
 
+    if ($data->{email_renewal_reminders}) {
```

Comment:
Could this (and 1538 similarly) be in `setup_garden_sub_params` so only needs to be in the one place? I know it's also called for cancellation etc but this check would still be there.

Reply:
I think `setup_garden_sub_params` is called before we have the report we can attach the extra field to. Not sure there's any good alternative.